aten/src/ATen/native/mps/operations/Randperm.mm

```diff
@@ -0,0 +1,80 @@
+//  Copyright © 2022 Apple Inc.
+
+#include <ATen/native/mps/OperationUtils.h>
+#include <ATen/native/TensorFactories.h>
+
+namespace at {
+namespace native {
+
+Tensor& randperm_out_mps(int64_t n, c10::optional<Generator> generator, Tensor& result) {
```

Comment:
Why is this not in Distribution.mm? I think a new file for this small op is overkill. Are we planning to add more ops to it?

Reply:
Usually I go for the way CPU/CUDA named their files for consistency